reservations/models/reservation.py

```diff
@@ -212,6 +210,14 @@ class Meta:
     def __str__(self) -> str:
         return f"{self.name} ({self.type})"
 
+    @property
+    def price_net(self) -> Decimal:
+        return round_decimal(self.price / (1 + self.tax_percentage_value / Decimal(100)), 6)
```

Comment:
I think we can round to the usual 2 decimal places when this is now an ephemeral and no longer a stored value.